server/server.go

```diff
@@ -455,13 +455,14 @@ func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag,
 		return nil
 	}
 
-	msg := fmt.Sprintf("Tag %s created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", tag)
+	username := slashCommand.Username
+	msg := fmt.Sprintf("@%s\nTag %s created by. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", username, tag)
```

Comment:
I was thinking we include the username inside the backticks as we don't want to mention the user. So something like:
```
Username: `@helloworld`
Tag %s created by. Waiting...
```

Also this will serve as a basis for the other additions such as https://github.com/mattermost/matterbuild/pull/45